angrmanagement/ui/dialogs/preferences.py

```diff
@@ -179,9 +179,13 @@ def _init_widgets(self):
         log_format_lbl = QLabel("Log datetime Format String:")
         log_format_lbl.setSizePolicy(QSizePolicy(QSizePolicy.Fixed, QSizePolicy.Fixed))
         log_format_layout.addWidget(log_format_lbl)
-        self.log_format_entry = QLineEdit(self)
-        self.log_format_entry.setClearButtonEnabled(True)
-        self.log_format_entry.setText(Conf.log_timestamp_format)
+
+        self.log_format_entry = QComboBox(self)
+        fmt: str = Conf.log_timestamp_format
+        for i in {fmt, "%X", "%c"}:  # pylint: disable=use-sequence-for-iteration
+            self.log_format_entry.addItem(i)
```

Comment:
It would be nice to show a format example because I cannot remember what `%X` and `%c` mean in this context

Reply:
Implemented